javascript/javascript2/week2/main.js

```diff
@@ -0,0 +1,98 @@
+"use strict"
+console.log("Script loaded");
+
+let products = getAvailableProducts();
+const lovedProducts = document.getElementById("loved-prducts");
+let lovedProductsNo = parseInt(lovedProducts.innerHTML);
+// after clicking the 'like_product' button, remove it
+function addProduct(event) {
```

Comment:
```suggestion
function addProduct(product) {
```
This function should probably just take a product and then add that product. Then the function can easier be reused